NeoMathEngine/src/GPU/CUDA/CudaMathEngineDnnDropout.cu

```diff
@@ -28,13 +28,29 @@ limitations under the License.
 
 namespace NeoML {
 
-void CCudaMathEngine::Dropout( const CDropoutDesc& dropoutDesc, const CFloatHandle& inputData, const CFloatHandle& outputData )
+CCudaMathEngineDropoutDesc::CCudaMathEngineDropoutDesc( IMathEngine& mathEngine, float rate, bool isSpatial,
+		bool isBatchwise, const CBlobDesc& input, const CBlobDesc& output, int seed ) :
+	Input(input),
+	Output(output),
+	ForwardRate(1.f - rate),
+	IsSpatial(isSpatial),
+	IsBatchwise(isBatchwise),
+	seed(seed) {};
```

Comment:
_minor_ _codestyle_
No need for `;`. The empty body should be located on a new line.
But it's your choice whether to write it like
```
    seed(seed)
{}
```
or 
```
    seed(seed)
{
}
```